src/features/connect/connect-button.tsx

```diff
@@ -32,23 +38,43 @@ const ConnectButtonInner = observer(
 
     return (
       <>
-        <Density sparse>
-          {providerOrigins.length === 0 ? (
-            <Button
-              icon={Wallet2}
-              actionType={actionType}
-              onClick={() =>
-                window.open('https://praxwallet.com/', '_blank', 'noopener,noreferrer')
-              }
-            >
-              Install Prax
-            </Button>
-          ) : (
-            <Button icon={Wallet2} actionType={actionType} onClick={onConnectClick}>
-              Connect wallet
-            </Button>
-          )}
-        </Density>
+        {variant === 'default' ? (
+          <Density sparse>
```

Comment:
suggestion (non-blocking): 
```suggestion
          <Density variant={variant === 'default' ? 'sparse' : 'compact'}>
```

Since https://github.com/penumbra-zone/web/pull/1942, it is possible to provide dynamic `variant` prop to Density, to not duplicate the code